### DISCUSSION TITLE
HVAC use case support

### DISCUSSION THREAD

**Discussion by heavyweight87** [enbility/eebus-go#95] [General]
Hi

Great work on this library! 

I wondered if you had any plans for HVACs?
I’m specifically interested in OHPCF

If you have no current plans how difficult do you think it would be to support the use case? Is it just a matter of providing an appropriate usecase.go file? It seems that your spine lib has everything needdd to support this

**Comment by DerAndereAndi** [comment]
Hi,

thanks for your question! :) I currently don't have plans to add that usecase myself, but feel free to create a PR :)
Please note that the data structure of OHPCF model is quite to all others. It combines descriptions, constraints and data in a single data structure. Therefor merges are not yet supported for this, and the system has to be extended to support it. So partial support for the according features are not announced and for each incoming notification, a full read is automatically triggered. As a consequence, it is currently not possible to easily identify which data items changed.

So there is no blocker existing, but also not all possible features supported out of the box.

Hope this helps for a starter
Andreas

**Comment by heavyweight87** [comment]
I would love to use this lib to control heatpumps, so there are actually a few usecases that are missing. Ill take a look to see what it will take to add them... 
For now I have a controlbox <-> CEMS <-> Heatpump with LPC and it works pretty great! Maybe ill add the heatpump to the examples (it only supports LPC)

**Reply by andig** [reply]
@heavyweight87 out of curiosity: what is CEMS? And which heatpump actually supports EEBus with a decent implementation?

**Reply by heavyweight87** [reply]
CEMS = Customer energy management system. There are a few... check out Vaillant for example

**Comment by heavyweight87** [comment]
Ill close this - I am working on adding more heatpump usecases
